src/Clock.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from 'react';
+
+type Props = {
+  clockName: string,
+};
+
+export const Clock: React.FC<Props> = ({ clockName }) => {
+  const [today, setToday] = useState(new Date());
+  const [prevClockName, setPrevClockName] = useState(clockName);
+
+  useEffect(() => {
+    const timerId = window.setInterval(() => {
+      setToday(new Date());
+      // eslint-disable-next-line no-console
+      console.info(new Date().toUTCString().slice(-12, -4));
```

Comment:
and use it i.e. here (could also in the `today` state)
```suggestion
      console.info(getTime());
```